packages/contracts/src/guise/LibGuiseLevel.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.17;
+
+import { IUint256Component } from "@latticexyz/solecs/src/interfaces/IUint256Component.sol";
```

Comment:
this file doesn't work and looks unfinished
in particular both pragma and the old import are invalid, I assume u didn't test this before pushing?